pkg/apis/scorecard/v1alpha1/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 The Operator-SDK Authors
```

Comment:
The current scorecard API version is v1alpha2. This "new" one is v1alpha1. Shouldn't it be v1alpha3?

Reply:
there was a change in the group with this version, so that led us to choose v1alpha1 for this.

Reply:
In that case it may be a good idea to copy code from `pkg/apis/scorecard/v1alpha2` to `pkg/apis/osdk/v1alpha2` and deprecate the original code in `pkg/apis/scorecard/v1alpha2`, so as to follow a  group path format. Thoughts?

Reply:
The new API is coming with changes to the structs to support kuttl . So, there is a story coming to update the code to use the new API in the next sprint. I believe thats when the v1alpha2 will be replaced with v1alpha1 .

Reply:
Sure, I'm not saying deprecate `v1alpha2` I'm saying deprecate its current path. I expect people to still be using the old scorecard for a while until `alpha scorecard` is out of alpha. However this change isn't strictly necessary although it would differentiate types properly.

Reply:
so if I understand, the current scorecard api code would move to /pkg/apis/osdk/v1alpha2, and the new scorecard api would move into /pkg/apis/scorecard/v1alpha1?  seems reasonable to me if thats the case.

Reply:
Yeah, and we'd follow the proper deprecation process and leave behind the current path `pkg/apis/scorecard/v1alpha2` in a deprecated state.

Reply:
@estroz @joelanford @jmccormick2001 
As per slack conv:
 Maybe to avoid the problem we use the same domain and call it v1alpha3 for now? We can change the domain later prior to 1.0. I suspect we'll do a full scrub across all of SDK to normalize domains and make them consistent.